cmd/openshift-install/create.go

```diff
@@ -249,7 +249,8 @@ func logComplete(directory string) error {
 		return err
 	}
 	kubeconfig := filepath.Join(absDir, "auth", "kubeconfig")
-	logrus.Infof("Install complete! Run 'export KUBECONFIG=%s' to manage your cluster.", kubeconfig)
-	logrus.Info("After exporting your kubeconfig, run 'oc -h' for a list of OpenShift client commands.")
+	// TODO: Direct users to web-console
```

Comment:
open github issues tracking these required changes, rather than TODOs in code... ?

Reply:
will do